src/index.js

```diff
@@ -42,13 +42,30 @@ export default class Users {
     app.get('templater').template('user-profile', 'ejs', __dirname+"/../views/profile.ejs")
     app.get('templater').template('user-forgot-email', 'ejs', __dirname+"/../views/forgot-email.ejs")
 
-    this.controllers.api = new APIController(app)
+    app.get('admin-ui').adminModel("user", {
```

Comment:
So getting rid of the try/catch for admin not being installed by the silent provide - once we make Application warn for un-gathered provides this will warn, good/bad (we want users to take those warnings seriously, but here it will be "intentional")